kieserver/62/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerBasicMulltiContainerTest.java

```diff
@@ -47,7 +49,10 @@
                 @TemplateParameter(name = "KIE_SERVER_PASSWORD", value = "${kie.password:Redhat@123}")
         }
 )
-public class DecisionServerBasicMulltiContainerTest extends DecisionServerBasicTest {
+@OpenShiftResources({
+        @OpenShiftResource("https://raw.githubusercontent.com/${template.repository:jboss-openshift}/application-templates/${template.branch:master}/secrets/decisionserver-app-secret.json")
+})
+public class DecisionServerBasicMulltiContainerTest extends DecisionServerTestBase {
```

Comment:
There was a typo already in the name of this class. Maybe take the chance to fix it?